README.md

```diff
@@ -72,11 +70,62 @@ We are in favor of [Test Driven Bug Fixing (TDBF)](https://geeknarrator.com/2018
 7. Add tag, which should trigger CI and publish binaries
 8. Merge release branch to develop
 
-Other resources
-----------------
+# Using iPXE
+
+This option assume that your apuX is in the same networks as your PC. Your PC
+in this case is used as HTTP and NFS server, which will be utilized to boot
+apuX over iPXE.
+
+```
+git clone https://github.com/3mdeb/pxe-server.git
+cd pxe-server
+NFS_SRV_IP=<your_ip> ./init.sh
+./start.sh
+```
+
+Please note that you may have NFS server running on host what leads to ports
+conflicts.
+
+After starting NFS and HTTP you can boot apuX. Please enable network booting
+using [sortbootorder](https://github.com/pcengines/sortbootorder#theory-of-operation).
+
+```
+iPXE> ifconf net0
+iPXE> dhcp net0
+iPXE> chain http://<your_ip>:8000/menu.ipxe
+```
+
+Choose `Debian stable netboot 4.14.y` after boot login (`[root:debian]`) and
+run:
+
+```
+flashrom -p internal -w apuX_x.y.z.rom
```

Comment:
@pietrushnic remember about apu1 exception where user has to provide chip explicitly